<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1480</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1691</y>
        <width>1435</width>
        <height>2902</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_printer">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ustawienia drukowania</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_printer">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="checkBox_printLabels">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Drukowanie etykiet</string>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_everyOtherRope">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Drukuj etykietę co drugą linkę</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_tip">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>(gdy wyłączone etykieta będzie drukowana tylko na koniec zadania)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_winding">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ustawienia zwijania</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_winding">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_startLength">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Długość początkowa (0-2000 [mm])</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_startLength">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>32</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox::up-button {
    width: 60%;
}
QSpinBox::down-button {
    width: 60%;
}</string>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_startLength">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_stopOffset">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Margines długości przed zatrzymaniem (0-1000 [mm])</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_stopOffset">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>32</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox::up-button {
    width: 60%;
}
QSpinBox::down-button {
    width: 60%;
}</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_stopOffset">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_guillotine">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ustawienia gilotyny</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_guillotine">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_upTime">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Czas podnoszenia gilotyny (1-10 [s])</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_upTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>32</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox::up-button {
    width: 60%;
}
QSpinBox::down-button {
    width: 60%;
}</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_upTime">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_downTime">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Czas opuszczania gilotyny (1-10 [s])</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_downTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>32</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox::up-button {
    width: 60%;
}
QSpinBox::down-button {
    width: 60%;
}</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_downTime">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_reset">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ustawienia przywracania bębna do pozycji zerowej</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_reset">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_searchTime">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Czas szukania punktu zero (2-20 [s])</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_searchTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>32</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox::up-button {
    width: 60%;
}
QSpinBox::down-button {
    width: 60%;
}</string>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_searchTime">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ui">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ustawnia interfejsu</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_ui">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QCheckBox" name="checkBox_buzzerSignals">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Sygnały dźwiękowe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_confirmationTime">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Czas potwierdzania następnej linki (1-10 [s])</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_confirmationTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>32</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox::up-button {
    width: 60%;
}
QSpinBox::down-button {
    width: 60%;
}</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_confirmationTime">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="text">
         <string>Zapisz zmiany</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Anuluj</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_startLength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_startLength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_startLength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_startLength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_confirmationTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_confirmationTime</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_confirmationTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_confirmationTime</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_searchTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_searchTime</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_searchTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_searchTime</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_downTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_downTime</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_downTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_downTime</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_upTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_upTime</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_upTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_upTime</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_stopOffset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_stopOffset</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_stopOffset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_stopOffset</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
